doc/mgr/dashboard.rst

```diff
@@ -127,62 +127,67 @@ The Ceph Dashboard offers the following monitoring and management capabilities:
 Overview of the Dashboard Landing Page
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Displays overall cluster status, performance, and capacity metrics. Shows instant
-feedback for changes in the cluster and provides easy access to subpages of the
-dashboard.
+The landing page of Ceph Dashboard serves as the home page and features metrics
+such as the overall cluster status, performance, and capacity. It provides real-time
+updates on any changes in the cluster and allows quick access to other sections of the dashboard.
+
+.. image:: dashboard-landing-page.png
+
+
+.. note::
+  You can change the landing page to the previous version from:
+  ``Cluster >> Manager Modules >> Dashboard >> Edit``.
+  Editing the ``FEATURE_TOGGLE_DASHBOARD`` option will change the landing page, from one view to another.
```

Comment:
Sorry for the missunderstanding @epuertat, when I first recorded it that was the case, since then I updated it to show it as `FEATURE_TOGGLE_DASHBOARD` to try to keep the consistentency with the naming and only use the `_v3` for internal naming.

And in the end the route `/dashboard` corresponds to the Landing Page of the Dashboard, so it seemed appropiate to keep the naming as just that.